lineapy/system_tracing/_op_stack.py

```diff
@@ -38,8 +66,15 @@ class Frame(Structure):
     ) + Frame._fields_
 
 PTR_SIZE = sizeof(POINTER(py_object))
-F_VALUESTACK_OFFSET = sizeof(Frame) - 2 * PTR_SIZE
-F_STACKTOP_OFFSET = sizeof(Frame) - PTR_SIZE
+F_VALUESTACK_OFFSET = sizeof(Frame) - 3 * PTR_SIZE
+if sys.version_info < (3, 10):
+    F_STACKTOP_OFFSET = sizeof(Frame) - 2 * PTR_SIZE
+elif (3, 11) > sys.version_info >= (3, 10):
+    F_STACKDEPTH_OFFSET = sizeof(Frame) - PTR_SIZE
+else:
+    raise NotImplementedError(
+        "LineaPy currently does not support Python 3.11 and above"
+    )
```

Comment:
Again, I don't think an user facing error should be raised here (we can log it internally as a bug, b/c we shouldn't get here given previous checks.).